app/src/main/java/org/simple/clinic/registration/facility/RegistrationFacilitySelectionScreenEvents.kt

```diff
@@ -27,3 +28,7 @@ data class RegistrationFacilityUserLocationUpdated(val location: LocationUpdate)
 data class RegistrationFacilityLocationPermissionChanged(val result: RuntimePermissionResult) : UiEvent {
   override val analyticsName = "Registration:Facility Selection:Location Permission Changed to $result"
 }
+
+data class RegistrationFacilityConfirmed(val facilityUuid: UUID) : UiEvent {
+  override val analyticsName: String = "Registration:Confirm Facility Selection"
```

Comment:
We format all the analytics events within a screen in a specific way. Can you follow that convention, please? This is missing the `:Facility Selection:` part.